downstream/modules/hub/proc-basic-repo-sync.adoc

```diff
@@ -13,7 +13,7 @@ All collections in the configured remote are downloaded to your custom repositor
 +
 [NOTE]
 ====
-To limit repository synchronization to specific collections within a remote, you can identify specific collections to be pulled using a requirements.yml file. See xref:proc-create-remote[Create a remote] for more information.
+To limit repository synchronization to specific collections within a remote, you can identify specific collections to be pulled using a requirements.yml file. See xref:proc-create-remote_remote-management[Create a remote] for more information.
```

Comment:
I'm assuming the additional _remote-management is the {Context}?

Reply:
Hey @ianf77 , Correct! Those files appear in this document several times now and that xref was causing issues. I've noted it in the tracker, so we can go ahead and merge this.